specs/api-spec.yaml

```diff
@@ -19,7 +19,7 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/RollbackReleaseResponse"
-  /orchestrator/application/rollback:
+  /orchestrator/application/rollback/deprecated:
```

Comment:
deprecated?

Reply:
removing this, as two apis merged in one